contracts/strategies/morpho/MetaMorphoStrategy.sol

```diff
@@ -324,15 +324,25 @@ contract MetaMorphoStrategy is ERC4626Strategy {
     uint256 _supplyQueueLen,
     uint256 _add
   ) internal view returns (uint256) {
+    uint256 _assetsSuppliedByVault;
```

Comment:
i'm not sure but maybe if you compile via ir there's no need for such optimization